### ISSUE TITLE
DST Staging Review (Consider): Design System guidance may mislead VFS teams

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#2834]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Button Icon
**Staging Review ticket:** [Staging Review: Design System Button Icon Component](https://github.com/department-of-veterans-affairs/va.gov-team/issues/82571)

## Findings details
The details of this ticket are for consideration only. No Staging Review issue was found.
**High-priority:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

As currently coded, the component only has three valid values for the `button-type` prop, and that prop determines what the button label will be. Since content is so constrained, does it make sense for the Content Considerations section on the Design System website to be providing guidance? That section implies that content is within a VFS team's control, when it's not. Other constrained components explicitly call out those limitations, eg. the [Crisis Line Modal](https://design.va.gov/components/modal/crisis-line-modal#content-considerations).

## Recommended action
If there's still value in having the Content Considerations guidance, maybe include some explanation in that section that there are only 3 supported states, and that this is the reasoning behind the label text for those states. As currently presented, it's easy to imagine a designer believing they can write their own text following these content guidelines --- only for their engineer to tell them nope, not supported.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
